src/docs/release/breaking-changes/image-cache-and-provider.md

```diff
@@ -0,0 +1,140 @@
+---
+title: [Changes to ImageCache and ImageProvider]
```

Comment:
are these supposed to be in `[]`? The one other guide I checked didn't have that...

Reply:
Done